prow/build_test_update.md

```diff
@@ -59,22 +59,84 @@ kubectl apply -f prow/cluster/hook_deployment.yaml
 ## How to test a ProwJob
 
 The best way to go about testing a new ProwJob depends on the job itself. If the
-job's test container can be run locally that is typically the best way to
-initially test the job because local debugging is easier than debugging a job in
-CI.
-
-Actually running the job on Prow is the next step. Before Prow can run your job,
-you'll need to supply the job's config. Typically, new presubmit jobs
-are configured to `skip_report`ing to GitHub and may not be configured to 
-automatically run on every PR with `always_run: true`. Once the job is stable
-these values can be changed to make the job run everywhere and become visible
-to users by posting results to GitHub (if desired).
-Changes to existing jobs can be trialed on canary jobs.
+job can be run locally that is typically the best way to initially test the job
+because local debugging is easier and safer than debugging in CI. See
+[Running a ProwJob Locally](#running-a-prowjob-locally) below.
+
+Actually running the job on Prow by merging the job config is the next step.
+Typically, new presubmit jobs are configured to `skip_report`ing to GitHub and
+may not be configured to automatically run on every PR with `always_run: true`.
+Once the job is stable these values can be changed to make the job run everywhere
+and become visible to users by posting results to GitHub (if desired). Changes
+to existing jobs can be trialed on canary jobs.
+
+ProwJobs can also be manually triggered by generating a YAML ProwJob CRD resource
+with mkpj and deploying that directly to the Prow cluster, however this pattern
+is generally not recommended. It requires the use of direct prod cluster access,
+allows ProwJobs to run in prod without passing presubmit validation, and can result
+in malformed ProwJobs CRDs that can jam some of Prow's core service components.
+See [How to manually run a given job on Prow](#how-to-manually-run-a-given-job-on-prow)
+below if you need to do this.
 
 ### Running a ProwJob Locally
 
-[Phaino](/prow/cmd/phaino) lets you interactively mock and run the job locally on your workstation
-in a docker container. Detailed instructions can be found in Phaino's [Readme](/prow/cmd/phaino/README.md)
+#### Using pj-on-kind.sh
+[pj-on-kind.sh] is a bash script that runs ProwJobs locally as pods in a [Kind] cluster.
+The script does the following:
+1. Installs [mkpj], [mkpod], and [Kind] if they are not found in the path. A [Kind]
+cluster named `mkpod` is created if one does not already exist.
+1. Uses [mkpj] to generate a YAML ProwJob CRD given job name, config, and git refs (if applicable).
```

Comment:
What is a CRD?
(CRD is mentioned earlier in the doc, but it would make the most sense to explain the acronym here)